Data race during TestWriteUseValidPair

```
=== RUN   TestWriteUseValidPair
==================
WARNING: DATA RACE
Write at 0x00c0001f44b0 by goroutine 57:
  github.com/pion/ice/v2.TestWriteUseValidPair()
      /home/travis/gopath/src/github.com/pion/ice/connectivity_vnet_test.go:612 +0xc7e
  testing.tRunner()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:909 +0x199

Previous write at 0x00c0001f44b0 by goroutine 80:
  github.com/pion/ice/v2.TestWriteUseValidPair.func1()
      /home/travis/gopath/src/github.com/pion/ice/connectivity_vnet_test.go:552 +0x15a
  github.com/pion/transport/vnet.(*Router).processChunks()
      /home/travis/gopath/pkg/mod/github.com/pion/transport@v0.10.0/vnet/router.go:447 +0x26b
  github.com/pion/transport/vnet.(*Router).Start.func1()
      /home/travis/gopath/pkg/mod/github.com/pion/transport@v0.10.0/vnet/router.go:224 +0x53

Goroutine 57 (running) created at:
  testing.(*T).Run()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:960 +0x651
  testing.runTests.func1()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:1202 +0xa6
  testing.tRunner()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:909 +0x199
  testing.runTests()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:1200 +0x521
  testing.(*M).Run()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:1117 +0x2ff
  main.main()
      _testmain.go:284 +0x347

Goroutine 80 (running) created at:
  github.com/pion/transport/vnet.(*Router).Start()
      /home/travis/gopath/pkg/mod/github.com/pion/transport@v0.10.0/vnet/router.go:221 +0x129
  github.com/pion/ice/v2.TestWriteUseValidPair()
      /home/travis/gopath/src/github.com/pion/ice/connectivity_vnet_test.go:572 +0x51a
  testing.tRunner()
      /home/travis/.gimme/versions/go1.13.linux.amd64/src/testing/testing.go:909 +0x199
==================
--- FAIL: TestWriteUseValidPair (0.41s)
    testing.go:853: race detected during execution of test
```

--- a/connectivity_vnet_test.go
+++ b/connectivity_vnet_test.go
@@ -549,7 +549,7 @@ func TestWriteUseValidPair(t *testing.T) {
 			m := &stun.Message{
 				Raw: c.UserData(),
 			}
-			if err = m.Decode(); err != nil {
+			if decErr := m.Decode(); decErr != nil {
 				return false
 			} else if m.Contains(stun.AttrUseCandidate) {
 				return false

